src/examples/source/service/workflow.grpc.pb.cc

```diff
@@ -0,0 +1,247 @@
+// Generated by the gRPC C++ plugin.
```

Comment:
As these files are generated, is it worth putting them in the repository? at first, I would say no, but I don't have a strict opinion on this. But if we put them in our repository, how is it generated? if it's generated by hand, would be nice to document in an appropriate place (maybe inline documentation in the .proto file would be a convenient location)

Reply:
A short notice on how to generate them is now included in the proto file.

Reply:
These files are need for the SmartPeakServer and SmartPeakGUI.